click-stream-dashboard/Makefile

```diff
@@ -0,0 +1,56 @@
+include ../.env
+
+list:
+	@curl ${ST_CONTROLLER}/schemas \
```

Comment:
Consider using httpie for requests. as well as running those In a container like 

docker run -it --rm --net=host clue/httpie